packages/app/src/app/components/Preview/DevTools/Terminal/index.tsx

```diff
@@ -174,11 +127,12 @@ class TerminalComponent extends React.Component<Props, State> {
   };
 
   render() {
-    const { height, hidden } = this.props;
+    const { height, hidden, theme } = this.props;
+    const { selectedShell } = this.state;
 
     return (
-      <div className={!hidden && 'terminal'} ref={this.setupResizeObserver}>
-        {!hidden && this.state.shells.length > 0 && (
+      <div className={!hidden && 'terminal'}>
+        {!hidden && this.state.shells.length !== 0 && (
```

Comment:
@CompuIves `length` can't be beneath `0` so the `> 0` was correct, no? 🤔

```suggestion
        {!hidden && this.state.shells.length > 0 && (
```